src/components/Home/index.js

```diff
@@ -117,6 +139,13 @@ export default class Home extends React.Component {
     })
   }
 
+  selectTownhall(townhall) {
+    this.setState({
+      selectedTownHall: townhall
+    });
+    $('.event-modal').modal('show');
```

Comment:
We would like to be able to get rid of jquery, so can you re implement this open functionality? Either just in the component, or use https://ant.design/components/modal/ from the ui library we're using (sorry I should have brought this up when we met before).